### ISSUE TITLE
Chatbot (UI): sentencepiece missing from requirements.txt (I think?)

### ISSUE THREAD

**Issue by one-lithe-rune** [nod-ai/SHARK#1424]
Trying to run off of main branch -- okay, my own branch rebased off of main -- atm (because I'm a mad fool obviously), and startup fails with:

```
(shark.venv) PS C:\develop\shark> python .\apps\stable_diffusion\web\index.py --device_allocation=caching --max_length=77 --output_dir="C:\develop\shark-exe\"
shark_tank local cache is located at C:\Users\skapusniak\.local/shark_tank/ . You may change this by setting the --local_tank_cache= flag
vulkan devices are available.
cuda devices are not available.
C:\develop\shark\shark.venv\Lib\site-packages\gradio\components.py:153: UserWarning: Unknown style parameter: columns
  warnings.warn(f"Unknown style parameter: {key}")
C:\develop\shark\shark.venv\Lib\site-packages\gradio\components.py:153: UserWarning: Unknown style parameter: object_fit  warnings.warn(f"Unknown style parameter: {key}")
C:\develop\shark\shark.venv\Lib\site-packages\diffusers\models\cross_attention.py:30: FutureWarning: Importing from cross_attention is deprecated. Please import from diffusers.models.attention_processor instead.
  deprecate(
Sucessfully loaded the tokenizer to the memory
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████| 2/2 [00:06<00:00,  3.06s/it]
Traceback (most recent call last):
  File "C:\develop\shark\apps\stable_diffusion\web\index.py", line 62, in <module>
    from apps.stable_diffusion.web.ui.utils import create_custom_models_folders
    from apps.stable_diffusion.web.ui.stablelm_ui import stablelm_chat
  File "C:\develop\SHARK\apps\stable_diffusion\web\ui\stablelm_ui.py", line 18, in <module>
    from apps.language_models.scripts.sharded_vicuna_fp32 import (
  File "C:\develop\SHARK\apps\language_models\scripts\sharded_vicuna_fp32.py", line 405, in <module>
    tokenizer = AutoTokenizer.from_pretrained(path, use_fast=False)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\develop\shark\shark.venv\Lib\site-packages\transformers\models\auto\tokenization_auto.py", line 702, in from_pretrained
    return tokenizer_class.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\develop\shark\shark.venv\Lib\site-packages\transformers\utils\import_utils.py", line 1086, in __getattribute__
    requires_backends(cls, cls._backends)
  File "C:\develop\shark\shark.venv\Lib\site-packages\transformers\utils\import_utils.py", line 1074, in requires_backends
    raise ImportError("".join(failed))
ImportError:
LlamaTokenizer requires the SentencePiece library but it was not found in your environment. Checkout the instructions on the
installation page of its repo: https://github.com/google/sentencepiece#installation and follow the ones
that match your environment. Please note that you may need to restart your runtime after installation.
```

I did rebuild my venv, before that try, so presumably that's a missing dependency?

After doing `pip install sentencepiece` it got past that point and started normally.

**Comment by workpioupiou** [comment]
Had the same error on the latest release. Not in 720

**Comment by one-lithe-rune** [comment]
Fixed by 4731c1a835a3847c11fde5aaa62943acccbd0ba1
